package.json

```diff
@@ -26,6 +26,7 @@
     "fs-extra": "^8.1.0",
     "husky": "^3.1.0",
     "lerna": "^3.20.2",
+    "open-cli": "^5.0.0",
```

Comment:
Please open another PR for `open-cli` related changes.

Reply:
:+1:

Sorry this keeps slipping in.  `npm run coverage` fails for me because the command `open coverage/index.html` is only valid on MacOS.  I'll make separate PRs.